application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -151,15 +155,17 @@ public void onModalSubmitted(ModalInteractionEvent event, List<String> args) {
 
     private void transferFlow(ModalInteractionEvent event, String channelId, String authorId,
             String messageId) {
+
+        Function<ForumPost, WebhookMessageCreateAction<Message>> sendMessageToMod =
+                post -> event.getHook()
+                    .sendMessage("Transferred to %s"
```

Comment:
is that actually editing the "Bot is thinking..." message or sending a second message?

Reply:
its the same message